easybuild/easyconfigs/d/Dask-ML/Dask-ML-2024.4.4-foss-2023a.eb

```diff
@@ -0,0 +1,54 @@
+easyblock = 'PythonBundle'
+
+name = 'Dask-ML'
+version = '2024.4.4'
+
+homepage = 'http://ml.dask.org/'
+description = """
+Dask-ML provides scalable machine learning in Python using Dask alongside popular machine
+learning libraries like Scikit-Learn, XGBoost, and others.
+"""
+
+toolchain = {'name': 'foss', 'version': '2023a'}
+
+builddependencies = [('hatchling', '1.18.0')]
+
+dependencies = [
+    ('Python', '3.11.3'),
+    ('scikit-learn', '1.3.1'),
+    ('dask', '2023.9.2'),
+    ('numba', '0.58.1'),
+    ('SciPy-bundle', '2023.07'),
+]
+
+use_pip = True
+sanity_pip_check = True
+
+exts_list = [
+    ('setuptools-scm', '8.1.0', {
+        'sources': ['setuptools_scm-%(version)s.tar.gz'],
+        'checksums': ['42dea1b65771cba93b7a515d65a65d8246e560768a66b9106a592c8e7f26c8a7'],
+    }),  # sparse needs setuptools-scm>=8
```

Comment:
This is a bit dangerous, as this will become the setuptools-scm used by anything having Dask-ML as dependency. The solution in `SCRIPro-1.0.18-foss-2023a.eb` seems better. Can you try to use that trick in here as well?